sbatches/farmshare/singularity-jupyter.sbatch

```diff
@@ -15,19 +15,21 @@
 # $ bash start.sh singularity /path/to/container <args>
 
 PORT=$1
-NOTEBOOK_DIR=${2:-${SCRATCH}}
-CONTAINER=${3:-docker://vanessa/repo2docker}
+NOTEBOOK_DIR=/farmshare/scratch/users/$USER
+CONTAINER=/farmshare/home/classes/bioe/301p/ce/ce
```

Comment:
Here you should be defaulting to this - like the deleted example above (but with a different default).

Reply:
Updated as of 5bd500a